src/analysis/raster/qgsrastercalcnode.cpp

```diff
@@ -382,3 +382,42 @@ QgsRasterCalcNode *QgsRasterCalcNode::parseRasterCalcString( const QString &str,
   return localParseRasterCalcString( str, parserErrorMsg );
 }
 
+QStringList QgsRasterCalcNode::referencedLayerNames( const QString &str )
```

Comment:
Also let's modify this function to just be called as `node->referencedLayerNames()`